service/lib/agama/software/callbacks/media.rb

```diff
@@ -75,13 +80,13 @@ def media_change(error_code, error, url, product, current, current_label, wanted
 
           question = Agama::Question.new(
             qclass:         "software.medium_error",
-            text:           error,
-            options:        [:Retry, :Skip],
-            default_option: :Skip,
+            text:           title + separator + broken_system_warning(error),
```

Comment:
We need to evolve the question API, and that's true. However, I find this composition a bit tricky (especially because you rely later on CSS to make it look good).

With the current API, you could:

1. Add the missing information to the `data:` hash.
2. Build a specialized component on the React side that understands such dialogs.